Buildings/ThermalZones/Detailed/Examples/FFD/UsersGuide.mo

```diff
@@ -87,9 +101,20 @@ Type <code>make all</code>
 </li>
 </ul>
 <p>
-please note to make use you have the right of executing the <code>compile.bat</code> when using method 2 on Windows.
-To check the availability right click the file and choose <code>Properties</code>. Go to <code>Security</code> tab and edit if you don't have the
+please note to make use you have the right of executing the <code>compile.bat</code> 
```

Comment:
What does this sentence fragment mean? Please revise.

Reply:
I am not sure what the original author meant with this and don't think it is even needed. However I can't reach them and wonder whether I should just delete the sentence or reformulate based on my best guess.

Reply:
Then please reformulate it so users can understand it.

Reply:
I decided to delete the sentence.